components/edit-collective/sections/HostMetrics.js

```diff
@@ -1,268 +1,32 @@
-    host(slug: $slug) {
-      id
-      slug
-      hostFeePercent
-      platformFeePercent
-      isActive
-      plan {
-        id
-        name
-        hostFeeSharePercent
-      }
-      hostMetrics {
-        hostFees {
-          value
-          currency
-        }
-        platformFees {
-          value
-          currency
-        }
-        pendingPlatformFees {
-          value
-          currency
-        }
-        platformTips {
-          value
-          currency
-        }
-        pendingPlatformTips {
-          value
-          currency
-        }
-        hostFeeShare {
-          value
-          currency
-        }
-        pendingHostFeeShare {
-          value
-          currency
-        }
-        totalMoneyManaged {
-          value
-          currency
-        }
-      }
-    }
-  }
-`;
-
 const HostMetrics = props => {
-  const { locale } = useIntl();
-  const { loading, data, error } = useQuery(metricsQuery, {
-    context: API_V2_CONTEXT,
-    variables: { slug: props.collective.slug },
-  });
-
-  if (loading) {
-    return <Loading />;
-  } else if (error) {
-    return <MessageBoxGraphqlError error={error} maxWidth={500} m="0 auto" />;
-  } else if (data.host && !data.host.isActive) {
-    return (
-      <MessageBox withIcon type="error" maxWidth={400} m="0 auto">
-        <FormattedMessage id="host.onlyActive" defaultMessage="This page is only available for active fiscal hosts" />
-      </MessageBox>
-    );
-  }
-
-  const displayHostFees = data.host.hostFeePercent || data.host.hostMetrics.hostFees.value ? true : false;
-
-  const displayPlatformFees =
-    data.host.platformFeePercent ||
-    data.host.hostMetrics.platformFees.value ||
-    data.host.hostMetrics.pendingPlatformFees.value
-      ? true
-      : false;
-
-  const displayPlatformTips =
-    data.host.platformFeePercent === 0 ||
-    data.host.hostMetrics.platformTips.value ||
-    data.host.hostMetrics.pendingPlatformTips.value
-      ? true
-      : false;
-
-  const displayHostFeeShare =
-    data.host.hostMetrics.hostFeeSharePercent ||
-    data.host.hostMetrics.hostFeeShare.value ||
-    data.host.hostMetrics.pendingHostFeeShare.value
-      ? true
-      : false;
-
   return (
     <Fragment>
       <SettingsTitle contentOnly={props.contentOnly}>
         <FormattedMessage id="Host.Metrics" defaultMessage="Host Metrics" />
       </SettingsTitle>
 
       <Box>
-        <P>
+        <MessageBox type="info" withIcon>
           <FormattedMessage
-            id="Host.Metrics.Description"
-            defaultMessage="Host Metrics with current values. Pending fees are charged at the beginning of the following month."
+            defaultMessage="We have moved host metrics to a <Link>brand new dashboard</Link>. This section will be removed soon."
+            values={{
+              Link: getI18nLink({
+                as: Link,
+                href: `/${props.collective.slug}/dashboard/reports`,
```

Comment:
As per https://github.com/opencollective/opencollective/issues/4914, the host dashboard is now deprecated. Let's use the latest admin panel link:

```suggestion
                href: `/${props.collective.slug}/admin/reports`,
```